.github/workflows/prev-version-integration.yaml

```diff
@@ -2,14 +2,16 @@ name: Previous Version Integration Tests Workflow
 
 on:
   schedule:
-    - cron: '30 */2 * * *'
+    # More information on cron https://crontab.guru/
+    - cron: '0 1 * * *'
 
 env:
- SELENIUM_GRID_PROVIDER: saucelabs
- CLOUD_WATCH_METRIC: false
- TEST_TYPE: Github-Action
- SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
- SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+  SELENIUM_GRID_PROVIDER: saucelabs
```

Comment:
Indentation seems off, just note to check once.

Reply:
I think one or 2 spaces are the same. Change to 2 to be more consistent with other places.